.github/workflows/test-integration-template.yaml

```diff
@@ -317,6 +325,9 @@ jobs:
         env:
           TEST_CHART_FLOW: ${{ matrix.scenario.flow }}
           TEST_OPENSHIFT_POST_RENDER: ${{ inputs.camunda-helm-post-render }}
+          # Nightly integration test env vars
```

Comment:
Those are not only `Nightly integration test env vars` (as we now unified the flow for PRs too).
Let's just remove this comment (there is also another one down there).

Reply:
removed, good catch